tuneme/templates/core/section_page.html

```diff
@@ -15,7 +15,7 @@
 <div class="block {{self.get_effective_extra_style_hints}}">
 {% for section in self.sections %}
   <h5>{{section.title}}</h5>
-  {% with page=section.featured_articles|first %}
+  {% with page=section.featured_articles|first recent_article=section.articles|first %}
```

Comment:
I think it would be better if we replaced this with a `.exists()` check as it would prevent looking up both `page` and `recent_article` if `page` does exist.

```jinja
{% if section.feature_articles.exists %}
  {% with page=section.featured_articles|first %}
    ....
  {% endwith %}
{% else %}
  {% with recent_article=section.articles|first %}
    ...
  {% endwith %}
{% endif %}